tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc

```diff
@@ -1503,30 +1503,33 @@ Status MarkForCompilationPassImpl::FindCompilationCandidates() {
           continue;
         }
       }
+    }
 
-      bool to_continue = false;
-      for (auto s : absl::StrSplit(
-               GetTaoBridgeOptions()->op_type_clustering_black_list, ',')) {
-        if (s == node->type_string()) {
-          VLOG(2) << "Rejecting " << node->type_string()
-                  << " as op type black list";
-          to_continue = true;
-          break;
-        }
+    bool to_continue = false;
+    for (auto s :
+         absl::StrSplit(GetTaoBridgeOptions()->op_type_clustering_black_list,
+                        ',', absl::SkipEmpty())) {
+      if (s == node->type_string()) {
+        VLOG(2) << "Rejecting " << node->type_string()
+                << " as op type black list";
+        to_continue = true;
+        break;
       }
+    }
 
-      for (auto s : absl::StrSplit(
-               GetTaoBridgeOptions()->op_name_clustering_black_list, ',')) {
-        if (s != "" && node->name().find(s.data()) != std::string::npos) {
-          VLOG(2) << "Rejecting " << node->name() << " as op name black list";
-          to_continue |= true;
-          break;
-        }
+    for (auto s :
+         absl::StrSplit(GetTaoBridgeOptions()->op_name_clustering_black_list,
+                        ',', absl::SkipEmpty())) {
+      // if (s != "" && node->name().find(s.data()) != std::string::npos) {
+      if (absl::StrContains(node->name(), s)) {
```

Comment:
As far as I could remember, sometimes we fallback to `std` for compatibility on lower TF version which use also lower version absl.

Reply:
If this passes CI, then it will be fine right?